client/src/Components/App.jsx

```diff
@@ -36,6 +36,15 @@ function App({ }) {
       .catch((err) => console.error(err));
   }, []);
 
+  useEffect(() => {
+    axios.get(`products/:product_id/styles`, { params: { product_id: currentProd.id } })
```

Comment:
i would add an if statement here to prevent API calls when currentProd is first set to an empty object